daemon/oci_windows.go

```diff
@@ -332,6 +332,30 @@ func (daemon *Daemon) createSpecWindowsFields(c *container.Container, s *specs.S
 		s.Windows.CredentialSpec = cs
 	}
 
+	// Do we have any assigned devices?
+	if len(c.HostConfig.Devices) > 0 {
+		if isHyperV {
+			return errors.New("device assignment not support for HyperV containers")
+		}
+		if system.GetOSVersion().Build < 17717 {
+			return errors.New("device assignment not support on builds older than 17717")
```

Comment:
I think the general pattern for build errors would be `device assignment requires Windows builds 17717 or later`. Could you also add a `// TODO: Bump to final RS5 build number` comment to make it easier to find once RS5 is RTM.